src/components/FAQ/FAQ.module.css

```diff
@@ -0,0 +1,72 @@
+.mainContainer {
+    display: flex;
+    flex-direction: row;
+    gap: 5em;
+    padding: 4em 0em 4em 0em;
+}
+
+.contentWrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 5em;
+    padding: 4em;
+}
+
+.textContainer {
+    display: flex;
+    flex-direction: column;
+    gap: 2em;
+    max-width: 50%;
+    align-items: center;
+    justify-content: center;
+    text-align: left;
+}
+
+.buttonContainer {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+}
+
+@media screen and (max-width: 667px) {
+    .mainContainer {
+        display: flex;
+        flex-direction: column;
+        gap: 2em;
+        padding: 2em 1em 2em 1em;
+    }
+
+    .mainContainer img {
+        display: none;
+    }
+
+    .textContainer {
+        max-width: 100%;
+    }
+
+    .contentWrapper {
+        padding: 0em;
+    }
+}
+
+@media screen and (min-width: 668px) and (max-width: 1024px) {
```

Comment:
I think it's great that you put effort in fixing the media querys even though the project isn't quite finished. I tend to leave those for last an I never have time to make them, but it's such an important part of the code. Good job!